sklearn/model_selection/_search.py

```diff
@@ -598,7 +598,15 @@ def fit(self, X, y=None, groups=None, **fit_params):
             train/test set.
 
         **fit_params : dict of string -> object
-            Parameters passed to the ``fit`` method of the estimator
+            Parameters passed to the ``fit`` method of the estimator.
+
+            Depending on the value, a fit parameter may be split into multiple
+            CV groups along with `X` and `y`. For example, the `sample_weight`
+            parameter to :meth:`sklearn.linear_model.SGDClassifier.fit`
```

Comment:
I'm not sure there's a need to reference a specific estimator. Very many of them support sample weight. But you can reference the glossary with ```:term:`sample_weight` ```